kafkabp/config.go

```diff
@@ -99,6 +100,12 @@ type ConsumerConfig struct {
 	// or it might make things worse.
 	// You are advised to test before using non-empty rack id in production.
 	RackID RackIDFunc `yaml:"rackID"`
+
+	// Optional. Defaults to 5. Values are in seconds.
```

Comment:
```suggestion
	// Optional. Defaults to 5 seconds.
```

Reply:
(The nice thing about `time.Duration` is that it allows users to specify the value in the unit that makes sense in the context. In this case I think your original approach is better though. Sorry for the waffling – I didn't consider the implication of "bare" `time.Duration` values)

Reply:
It's all good, I'm learning :)